keyless/pepper/example-client-rust/src/lib.rs

```diff
@@ -439,6 +439,14 @@ async fn verify_firestore_pepper_entry(
         .await
         .unwrap();
 
+    // Wait for a short duration to ensure the firestore entry is available.
+    // This is required because firestore writes may be asynchronous.
+    utils::print(
+        "Waiting for 2 seconds to ensure the firestore entry is available...",
+        false,
+    );
+    tokio::time::sleep(std::time::Duration::from_secs(2)).await;
+
```

Comment:
is this because you experienced some delay during local testing?

Reply:
Yeah, now that the writes are asynchronous, the client example can flake because the pepper fetch call returns immediately, but the firestore emulator is only updated asynchronously. The sleep prevents any flakes 😄